src/containers/Dev/Snapshots.tsx

```diff
@@ -248,7 +230,17 @@ const Snapshots: React.FC = () => {
               className="clickable blue"
               onClick={async () => {
                 showToast({ title: 'Download started...', timeout: 2000 })
-                await downloadSnapshot(filename)
+                await downloadFile(
```

Comment:
Made a generic utility function "download file" so can be re-used in Document viewer (and anywhere else we might need it)